src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/CreateResourcesValidation.feature

```diff
@@ -226,7 +226,7 @@ Feature: Resources "Create" Operation validations
                     }
                   """
 
-        @API-160 @POST
+        @API-160 @API-233 @POST
         Scenario: 10 Create a document with an extra property (overpost) (Resource)
```

Comment:
The original commits duplicated this scenario in `DataStrictness`. I deleted the duplicate and put the new serial number here on this existing scenario.